src/patternfly/components/Button/docs/code.md

```diff
@@ -30,6 +30,8 @@ Semantic buttons and links are important for usability as well as accessibility.
 | `.pf-m-plain` | `.pf-c-button` | Modifies for icon styles. This button has no background or border, uses a standard text color, and is used for `.pf-m-plain` icon buttons such as close, expand, kebab, etc. |
 | `.pf-m-inline` | `.pf-c-button.pf-m-link` | Modifies for inline styles. This button is presented similar to a normal link, has no padding, and is displayed inline with other inline content. |
 | `.pf-m-block` | `.pf-c-button` | Creates a block level button. |
+| `.pf-m-control` | `.pf-c-button` | Modifies for control styles. **Note: ** This modifier should only be used when using buttons in the Input or ClipboardCopy components. |
+| `.pf-m-expanded` | `.pf-c-button.pf-m-input` | Modifies an input button to be expanded. |
```

Comment:
Maybe instead of "to be expanded" change to "for the expanded state" or something. Currently it reads like the button is expandable, instead of representing the expanded state of something else.